app/preferences/styles.ts

```diff
@@ -0,0 +1,108 @@
+'use client';
+
+import NextImage from 'next/image';
+import styled from 'styled-components';
+import COLORS from '../../styles/colors';
+
+export const Background = styled.main`
+  flex-direction: column;
+  min-width: 100%;
+  min-height: 100svh;
+  display: flex;
+  align-items: center;
+  justify-content: start;
+  background-color: ${COLORS.gray2};
+  overflow: hidden;
+`;
+
+export const InlineContainer = styled.main`
+  width: 30%;
+  height: 85%;
+  flex-direction: column;
+  margin-top: 2%;
+  margin-bottom: 2%;
+
+  @media (max-width: 1200px) {
+    width: 45%;
+  }
+  @media (max-width: 768px) {
+    width: 85%;
+  }
+`;
+
+export const Image = styled(NextImage)`
+  width: 20px;
+  height: 20px;
+  margin-bottom: -2px;
+`;
+
+export const Rectangle = styled.main<{
+  variant: 'light' | 'dark';
+  widthPercentage: string;
+}>`
+  width: ${({ widthPercentage }) => widthPercentage};
+  height: 7px;
+  display: inline-block;
+  background: ${({ variant }) =>
+    variant === 'light' ? COLORS.gray8 : COLORS.gray10};
+  border-radius: 2px;
+`;
+
+export const Container = styled.main`
+  width: 100%;
+  height: 53%;
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  margin: 25px 0px;
+  justify-content: space-between;
+  background-color: white;
+  border-radius: 8px;
+  padding: 13%;
+`;
+
+export const InputText = styled.text`
+  margin-top: 6%;
+`;
+
+export const Input = styled.input`
+  width: 100%;
+  height: 22px;
+  border-color: ${COLORS.gray4};
+  border-style: solid;
+  border-radius: 4px;
+`;
+
+export const Title = styled.h1`
+  font-size: 28px;
+  text-align: start;
+  color: ${COLORS.gray11};
+  margin-top: 0;
```

Comment:
delete `margin-top` and add `margin-bottom: 2.25rem;` to add more space between title and the first input field

Reply:
The reason I did margin-top: 0 because the title text has some default text. I think if do this I end up with  a lot of spacing btween the title and the rest of the stuff: 
<img width="595" alt="Screenshot 2024-10-19 at 5 35 28 PM" src="https://github.com/user-attachments/assets/f9541d2e-ee8f-47e8-a946-74b4916ebfd5">